tests/rbac/common/role/create_role_test.py

```diff
@@ -20,6 +20,7 @@
 from rbac.common import addresser
 from rbac.common import protobuf
 from rbac.common.role.role_manager import RoleManager
+
```

Comment:
don't include files unrelated to the PR

Reply:
If someone submits a PR without first running black to format their touched files then they will be touched in the next PR that is properly formatted using black.

Reply:
still, don't include files unrelated to the PR

Reply:
We'll make a separate PR soon to black non-compliant files and enforce black in Travis.